src/platforms/python/guides/pyspark/index.mdx

```diff
@@ -33,7 +33,7 @@ if __name__ == "__main__":
 
 ### Worker
 
-The spark worker integration is supported for Spark 2.4.0 and above.
+The spark worker integration is supported for Spark 2.4.x. It is **not supported for Spark 3**.
```

Comment:
I recommend either bolding the whole sentence "It is not supported..." or just bolding "not supported".